### DISCUSSION TITLE
Can I calculate values in Tinymce just like I calculate in excel?

### DISCUSSION THREAD

**Discussion by Amsilva89** [tinymce/tinymce#8637] [General]
I need calculate values in Tinymce, for example: "=sum(cellA1,cellA2...)".
It´s possible I doing that in Tiny?

**Comment by lostkeys** [comment]
@Amsilva89. No it is not possible to do in TinyMCE. This is a very complex feature for tables and we don't have any plans to introduce it right now. To help us better understand and prioritize your request, can you fill us in on the following details. If you don't want to do it publicly, feel free to submit your feedback over at our [roadmap website](https://www.tiny.cloud/roadmap/)

1. Why is the requested feature important for your app/service?
2. What is the consequence of not having the requested feature?
3. How important would you rate the requested feature choosing from nice-to-have, important or critical?

Thanks,
Fredrik – Product Manager
